html/elements/input/hidden.json

```diff
@@ -0,0 +1,56 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-hidden": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/hidden",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": null
```

Comment:
Is there a reason I'm missing for the `null` here? The original table has "Yes" and is seems very likely to be supported.